src/treetensornetworks/projttns/projttn.jl

```diff
@@ -68,3 +68,21 @@ function make_environment(P::ProjTTN, state::AbstractTTN, e::AbstractEdge)
   )
   return P
 end
+
+function projected_operator_tensors(P::ProjTTN)
+  environments = ITensor[environment(P, edge) for edge in incident_edges(P)]
+  # manual heuristic for contraction order fixing: for each site in ProjTTN, apply up to
+  # two environments, then TTN tensor, then other environments
+  if on_edge(P)
+    itensor_map = environments
+  else
+    itensor_map = Union{ITensor,OneITensor}[] # TODO: will a Hamiltonian TTN tensor ever be a OneITensor?
+    for s in sites(P)
+      site_envs = filter(hascommoninds(operator(P)[s]), environments)
+      frst, scnd, rst = _separate_first_two(site_envs)
+      site_tensors = vcat(frst, scnd, operator(P)[s], rst)
+      append!(itensor_map, site_tensors)
```

Comment:
Do you know what all of this is doing? It is pretty hard to understand. I thought this code would just need to loop through `sites(P)` and grab the operator on that site, i.e. `operator(P)[s]`.

Reply:
Yes, I think it's a heuristic hardcoded contraction order. If the optimal contraction order was determined afterwards, the code could look simpler.

Reply:
I see. Since it is a tree tensor network, it seems like a simpler way to write it would be to collect the tensors as a TTN like `region_operator_ttn = TTN([operator(P)[s] for s in sites(P)])` and then determine the order with `post_order_dfs_vertices(region_operator_ttn)`.

Reply:
I suggest we do this in another PR since it is not exactly related to ProjTTNSum?

Reply:
Sure, makes sense.

Reply:
I raised an issue about it here: #133.